packages/vehicle-rental-overlay/src/index.js

```diff
@@ -0,0 +1,251 @@
+import { divIcon } from "leaflet";
+import memoize from "lodash.memoize";
+import * as BaseMapStyled from "@opentripplanner/base-map/lib/styled";
+import { getCompaniesLabelFromNetworks } from "@opentripplanner/core-utils/lib/itinerary";
+import {
+  companyType,
+  mapSymbolsType,
+  stationsType
+} from "@opentripplanner/core-utils/lib/types";
+import FromToLocationPicker from "@opentripplanner/from-to-location-picker";
+import PropTypes from "prop-types";
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+import {
+  CircleMarker,
+  FeatureGroup,
+  Marker,
+  MapLayer,
+  Popup,
+  withLeaflet
+} from "react-leaflet";
+
+import { floatingBikeIcon, hubIcons } from "./bike-icons";
+import * as Styled from "./styled";
+
+const getStationMarkerByColor = memoize(color =>
+  divIcon({
+    className: "",
+    iconSize: [11, 16],
+    popupAnchor: [0, -6],
+    html: ReactDOMServer.renderToStaticMarkup(
+      <Styled.StationMarker color={color} />
+    )
+  })
+);
+
+class VehicleRentalOverlay extends MapLayer {
```

Comment:
Definitely deserves JSdoc given that it serves as the overlay for a number of different modes.